packages/scaffold-config/test/unit/fixtures.ts

```diff
@@ -0,0 +1,32 @@
+import { defineComponentFramework } from '../../src'
+
+export const solidDep: Cypress.CypressComponentDependency = {
+  type: 'solid-js',
+  name: 'Solid.js',
+  package: 'solid-js',
+  installer: 'solid-js',
+  description: 'Solid is a declarative JavaScript library for creating user interfaces',
+  minVersion: '^1.0.0',
+}
+
+// must be default export
+export const solidJs = defineComponentFramework({
```

Comment:
Anything preventing this from being `export default defineComponentFramework({...})`?